docs/platform/howto/list-vpc-peering.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Virtual private cloud (VPC) peering in Aiven
+sidebar_label: VPC peering
+---
+
+import DocCardList from '@theme/DocCardList';
+
+The VPC peering capability supported on the Aiven Platform improves network connectivity and security. It simplifies architecture, helps reduce network latency, and enhances resource sharing while maintaining isolation and control.
+
+[VPC](/docs/platform/concepts/vpcs) peering is a networking connection between two VPCs.
+It allows private and direct communication between the VPCs with no traffic routing over
+the public internet.
+
+### VPC peering characteristics
+
+- Private communication: Private IP addresses used by VPCs to communicate
+- High performance: Low latency thanks traffic remaining on the cloud provider's network
```

Comment:
```suggestion
- High performance: Low latency thanks to traffic remaining on the cloud provider's network
```

or 
```suggestion
- High performance: Low latency because traffic remains within the cloud provider's network
```